etc/galasa-tests/com.ibm.cics.cip.cbsa.galasa.tests.install/build.gradle

```diff
@@ -0,0 +1,52 @@
+
+// This section tells gradle which gradle plugins to use to build this project.
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+    id 'dev.galasa.tests' version '0.15.0'
```

Comment:
0.15.0 is quite old, you should be able to use the same version of Galasa that you're using for the tests (0.39.0)